WooCommerce/Classes/Authentication/Jetpack Setup/Site Credential Login/SiteCredentialLoginView.swift

```diff
@@ -118,7 +118,8 @@ struct SiteCredentialLoginView: View {
                                                                   text: $viewModel.password,
                                                                   isSecure: true,
                                                                   errorMessage: nil,
-                                                                  isFocused: focusedField == .password))
+                                                                 isFocused: focusedField == .password,
+                                                                 autocapitalization: .none))
```

Comment:
Nitpick: Indentation needs to be updated.

Reply:
5661c09d63b6d9ad450a3f089e39a7faeab2a9ec